docs/packaging/creating-a-new-package.md

```diff
@@ -104,4 +165,40 @@ Once the build completes, your directory should now include the following files:
 └── tree-2.1.1-1-1-x86_64.eopkg
 ```
 
-All these files _except_ the `*eopkg` file should be included when you submit the package for review.
+All these files _except_ the `*eopkg` file(s) should be included in your pull request. You will remove them after testing the package.
```

Comment:
"them" is ambiguous here; it could mean the eopkg files, or all of the other files.
```suggestion
All these files _except_ the `*eopkg` file(s) should be included in your pull request. You will remove the `.eopkg` files after testing the package.
```

Reply:
Fixed